book/src/tutorial/rust-std.md

```diff
@@ -0,0 +1,73 @@
+# Combining Rust `std` with `uefi`
+
+## TL;DR
+
+In Mid-2024, we recommend to stick to our normal guide. Use this document as
+guide and outlook for the future of UEFI and Rust.
+
+## About
+
+In the `uefi` crate, we provide tooling to create a `no_std` + `no_main`
+binary, i.e., an UEFI image, in a convenient way. However, there is also the
+option to create a "standard" binary with Rust. The result is the same, but the
+build process and the overall programming is much more similar to regular
+Rust binaries written for Operating Systems, such as Linux, Windows, or macOS.
+
+## Standard binary vs `no_std` binary
+
+A Rust standard binary is a binary that uses the Rust standard library. This
+means that the Rust standard library (_`std` implementation_) is implemented for
+the given platform. A `no_std` (strictly speaking, `no_std` + `no_main`) on the
+other hand can only use the `core` library, but not `std`. Thus, you need to
+write more functionality on your own or use libraries, such as the `uefi` crate.
+
+## About `std` for `uefi`
+
+Using `std` is always the recommended and standard way. However, the `std`
+implementation is not feature-complete and work in progress
+(Mid-2024, Rust stable 1.80), and will also be it in the coming months and
+probably years. Over time, you will need less and less specific features of
+`uefi` and can utilize standard and well-known OS-independent APIs in `std`
+instead. Just as you would to in the typical Linux application, for example.
+
+The `uefi` crate is suited to extend the functionality of these "standard
+binaries".
+
+## Code Example
+
+Please refer to [`<repo>/uefi-std-example`](/uefi-std-example/README.md) to
+see a specific example. The relevant `main.rs` looks as follows:
+
+```rust
```

Comment:
Rather than writing code directly in the doc, you can include the uefi-std-example code: https://rust-lang.github.io/mdBook/format/mdbook.html#including-files

That ensures the code in the book doesn't get out of date.

Reply:
thank you!